### DISCUSSION TITLE
Tooltips: option to display content of footer links

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#6403] [Ideas]
It could be a neat idea for the better tooltips feature to display the content of a footer when a user howers over the link.

This would enable wikipedia-like linking of references and could assist in keeping a good reading flow...

Not sure doable this would be, but I hope it can be considered.

**Comment by alexvoss** [comment]
Not sure what you man by "content of a footer" but this sounds similar to https://github.com/squidfunk/mkdocs-material/discussions/5727

**Reply by Andre601** [reply]
Yeah, like that, but for footnote links.

And I meant the content that a footnote[^1] contains, like the one shown at the bottom here. It would be nice if the better tooltips feature could in some way display them when hovering over the footnote in the text.

[^1]: I mean to display this text when someone hovers over the small "1" displayed in the text above.
